package.json

```diff
@@ -15,28 +15,19 @@
   "engines": {
     "npm": "please-use-pnpm",
     "yarn": "please-use-pnpm",
-    "node": "16",
+    "node": "18",
```

Comment:
Is this necessary for the migration? Some of our internal projects might still be on node 16 and I'm not sure how easily they can migrate to node 18.